homework-submission/todo_react/src/App.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import "./App.css";
+import Info from "./Info";
+import data from "./data";
+import TodoLists from "./TodoLists";
+import ItemCreate from "./ItemCreate";
+
+
+class App extends Component {
+  state = {
+    todos: data
+  };
+
+  handleClick = (id) => {
+    const newTodos = this.state.todos;
+
+    newTodos.forEach(todo => {
+      if (todo.id === id) {
+        todo.done = !todo.done;
+      }
+    });
+    this.setState({ todos: newTodos });
+  };
+  removeItem = (item) => {
+    let newState = this.state.todos
+    newState.splice(item, 1)
```

Comment:
The splice method is same as the previous comments, it does mutate the original Object, which is `state`.